tests/perf/utils/prometheus_metrics.go

```diff
@@ -0,0 +1,112 @@
+//go:build perf
+// +build perf
+
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"log"
+	"os"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/push"
+)
+
+type DaprMetrics struct {
+	Baseline_latency       float64
```

Comment:
In Go, convention is to avoid snake_case. We use:

- PascalCase for exported properties
- camelCase for non-exported properties


```suggestion
	BaselineLatency       float64
```

Please update the other properties too.

Reply:
Changed